apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
+                    <p class="text-center font-weight-bold">No course instances found.</p>
+                    <p class="text-center mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p class="text-center">
+                      To create one, click <strong>Add course instance</strong>.
```

Comment:
It might be nice to show an inline call to action here. For instance, that's what the [GitHub wiki pages](https://github.com/PrairieLearn/cla/wiki) does if there aren't any pages yet:

<img width="1111" alt="Screenshot 2024-12-10 at 09 28 49" src="https://github.com/user-attachments/assets/de6cc24b-f8ab-4e44-8321-e5d75ebdfcb7">

Reply:
Agreed -- I added this:

<img width="910" alt="image" src="https://github.com/user-attachments/assets/2192d7f8-9589-47e8-951a-7452fe116a46">

Reply:
Looks great 💯